packages/pwa-buildpack/lib/WebpackTools/plugins/ServiceWorkerPlugin.js

```diff
@@ -32,7 +30,7 @@ class ServiceWorkerPlugin {
         new WorkboxPlugin.GenerateSW(config).apply(compiler);
     }
 
-    configureInjectManifest() {
+    applyInjectManifest(compiler) {
```

Comment:
The functions in this file could really use JSDoc-style comments describing what they do.

Reply:
I understand this may be out of scope for the current PR.

Reply:
Yeah, you are right. Let me add those.